server/test/controllers/dev/FeatureFlagOverrideControllerTest.java

```diff
@@ -12,7 +12,7 @@
 
 public class FeatureFlagOverrideControllerTest extends ResetPostgres {
 
-  private static final String FLAG_NAME = "CF_OPTIONAL_QUESTIONS";
+  private static final String FLAG_NAME = "ALLOW_CIVIFORM_ADMIN_ACCESS_PROGRAMS";
```

Comment:
This is just happens to use this flag for the test. Changing it to one that will remain.